src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class FIgureSupplier {
```

Comment:
The class name 'FIgureSupplier' seems to be a typo; class names should be in CamelCase with the first letter of each word capitalized. It should be 'FigureSupplier'.